oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/MetadataRegistry.java

```diff
@@ -164,6 +178,20 @@ public MeasureModel registerMeasureModel(Model model, BanyanDBStorageConfig conf
             if (!tagsAndFields.fields.isEmpty()) {
                 throw new StorageException("index mode is enabled, but fields are defined");
             }
+        } else {
+            List<String> storageOnlyTags
+                = tagsAndFields.tags.stream()
+                                    // filter out index tags
+                                    .filter(tagMetadata -> !tagMetadata.isIndex())
+                                    // filter out seriesID tags
+                                    .filter(tagMetadata -> !seriesIDColumns.contains(tagMetadata.getTagSpec().getName()))
+                                    .map(tagMetadata -> tagMetadata.getTagSpec().getName())
+                                    .collect(Collectors.toList());
+            if (!storageOnlyTags.isEmpty()) {
+                throw new StorageException(model.getName() + ":"
+                                               + "storage only tags(" + storageOnlyTags + ") are not recommended in measure model, "
+                                               + "please change the entity to index mode or use field(@BanyanDB.MeasureField) instead");
+            }
```

Comment:
This part is the new logic. If there are storage-only tags exist in the installation stage, an exception will be thrown out.